index.html

```diff
@@ -0,0 +1,556 @@
+                  <a
+                    class="
+                      about-link
+                      nav-link
+                      active
+                      lato
+                      text-danger
+                      zoom
+                      link-secondary
+                    "
+                    aria-current="page"
+                    href="about.html"
+                    >About</a
+                  >
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link lato greyer fw-bolder lato" href="#program"
+                    >Program</a
+                  >
+                </li>
+                <li class="nav-item">
+                  <a
+                    class="nav-link lato greyer fw-bolder lato"
+                    href="#speakers"
+                    >Speakers</a
+                  >
+                </li>
+                <li class="nav-item">
+                  <a
+                    class="nav-link lato greyer fw-bolder lato"
+                    href="#partners"
+                    >Partners</a
+                  >
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link lato greyer fw-bolder lato" href="#"
+                    >Past</a
+                  >
+                </li>
+                <li class="nav-item">
+                  <button type="button" class="btn btn-outline-danger lato">
+                    Web Summit
+                  </button>
+                </li>
+              </ul>
+            </div>
+          </div>
+        </div>
+
+        <div
+          class="
+            text
+            container-fluid
+            col-md-8 col-lg-8 col-xl-8
+          "
+        >
+          <h3 class="bg-gradient-success lato">
+            “The best technology conference on the planet”- Forbes
+          </h3>
+          <h1 class="coco mb-4 crazy-text">
+            Web Summit 2022 – see you in Lisbon
+          </h1>
+          <p class="shadow-sm p-3 mb-5 bg-white rounded lato greyer">
+            Web Summit is the world's most grandiose conference bridging the
+            world of tech, media and government.T he expo consistently brings
+            prominent industry members and media personalities to Lisbon's
+            Altice Arena.
+          </p>
+
+          <h2 class="coco">2022.11.1(THU) ~ 4(FRI)</h2>
+          <h4 class="lato pb-5 m-0">
+            @ Web Summit 2022 dates – 1 to 4 November.
+          </h4>
+        </div>
+      </section>
+
+      <section id="program" class="container-fluid program m-0 p-0">
+        <h3 class="lato white col text-center pt-5">Main Program</h3>
+        <span class="line mb-5"></span>
+        <div class="column container py-1 col-md-7 col-lg-7 col-xl-7">
+          <div
+            class="
+              wrap-program
+              d-md-flex d-lg-flex d-xl-flex
+              justify-content-md-center
+            "
+          >
+            <div
```

Comment:
- [ ] Great job making the changes from the previous reviewer. Please notice that your text is breaking into other sections. which makes it not match the design prescribed. ensure that the spacings and margins are correct as per the reference design. 
## Text collapsing into each other
![image](https://user-images.githubusercontent.com/18730106/147390037-a6da1d69-74a9-4714-ad02-ca6496e6599b.png)

## Text breaking out of the container
![image](https://user-images.githubusercontent.com/18730106/147390063-aabba57a-439b-46be-85ab-66c85f0acb39.png)